RevenueCatUI/CustomerCenter/Views/WrongPlatformView.swift

```diff
@@ -92,13 +88,16 @@ struct WrongPlatformView: View {
                     }
                 }
             }
-            if let url = supportURL {
```

Comment:
removing this check will always show the button even if there's no `supportURL` configured right?

Reply:
Yeah, good point

Reply:
Yes, but if we want to show the alert there's no other alternative right?

Reply:
We only want to show the alert if there's a `supportURL`and someone taps the Contact Support button using a simulator. If there's no `supportURL` , which it's pretty common, the Contact Support button shouldn't be shown at all

Reply:
Ok so I've extracted `canOpenURL` out of the creation of the url, and check on tap if it can be opened at runtime 👍